### DISCUSSION TITLE
Some tailwind classes don't work/get picked up slowly

### DISCUSSION THREAD

**Discussion by xxAlin** [tailwindlabs/tailwindcss#9049] [Help]
Hi guys!

Trying to use tailwind for one of my projects. I've had some issues with it when it's not picking up certain values. For example it's not picking margin classes like "m-3, m-2" etc. But for some weird reason it picked up "m-5". I tried with "m-5" a few days ago and it wasn't working either and today it worked. Then I tried again "m-4" and it works and now going back to "m-5" or any other margin class doesn't work.

Is it like a processing issue or my setup? Im not sure what's happening.

**Comment by wongjn** [comment]
Sounds like you might need to take a look at your [content configuration](https://tailwindcss.com/docs/content-configuration) and ensure you have told Tailwind to scan the correct source files.

Otherwise if you are sure this is correct, we would need to look at your project (or a condensed version of it) to further diagnose the problem.
